.licenses/go/github.com/gofrs/uuid.dep.yml

```
@@ -0,0 +1,37 @@
+---
+name: github.com/gofrs/uuid
+version: v4.2.0+incompatible
+type: go
+summary: Package uuid provides implementations of the Universally Unique Identifier
+  (UUID), as specified in RFC-4122 and DCE 1.1.
```

```suggestion
  (UUID), as specified in RFC-4122 and the Peabody RFC Draft (revision 02).
```

You change it to the content from github.com/gofrs/uuid@v3.2.0, but this project is using github.com/gofrs/uuid@v4.2.0, so the license cache should contain the content from that version:

https://github.com/gofrs/uuid/blob/v4.2.0/uuid.go#L23